_data/gaming.yml

```diff
@@ -1,4 +1,10 @@
 websites:
+  - name: 2K Games
+    url: https://2k.com/
+    twitter: 2K
+    facebook: 2K
```

Comment:
I can't find any facebook page on https://facebook.com/2K

Reply:
This is very strange - it seems to work just fine on mobile and desktop when signed in. Otherwise, it gives a prompt to sign in on desktop, and a "page not found" on mobile.

FYI, Facebook is currently having an outage, if you're checking this right now.

Reply:
Okay I'll check again in an hour or so. Maybe it'll work then 🤷‍♂

Reply:
Facebook page [2K](https://facebook.com/2K) redirects to 2K.anz for me. 2K website links to [2kgame](https://www.facebook.com/2kgame/) and I get 
> Sorry, this content isn't available at the moment